package.json

```diff
@@ -43,11 +51,18 @@
     "@angular/platform-browser-dynamic": "^10.0.12",
```

Comment:
Probably should keep up to date angular. At least match version in weaver-components.

Reply:
We will upgrade in the next sprint.